word_guess.rb

```diff
@@ -0,0 +1,163 @@
+#word guess game
+
+#
+require "colorize"
+
+class Word_guess
+	attr_accessor :wrong_count, :word_list, :secret_word, :game_word, :game_word_array
+
+	def initialize
+		@wrong_count = -1
+		@word_list = nil
+		@secret_word = nil
+		@game_word = nil
+		@game_word_array = nil
+		@guessed_letters = []
+	end
+
+	def word_gen
+		@word_list = {
+		"celebrity" => ["sting", "beyonce", "madonna", "bono", "fabio", "nas", "prince", "adele","beck", "tupac"],
+		"difficult" => ["ghost","pacman","joker", "bricks", "grandmother", "ninja", "fuzziest", "jazzy", "dwarves", "buffalo"],
+		"animals" => ["monkey", "panda", "cat", "anteater", "girrafe", "elephant", "hotdog", "tiger"]
+		}
+
+		puts "Please pick a theme: celebrity, animals or difficult"
+		theme_input = gets.chomp.downcase
+		until theme_input == "celebrity" || theme_input == "difficult" || theme_input == "animals"
+			puts "Wrong. Please pick one of the following: celebrity, animals or difficult"
+			theme_input = gets.chomp.downcase
+		end
+
+		@secret_word = @word_list[theme_input].sample
+		@game_word = ("_"*@secret_word.length)
+
+		@secret_word_array = @secret_word.split(//)
+		@game_word_array = @game_word.split(//)
+	end
+
+	#asks user for a letter to guess in the word
+	def user_input
+		puts "Please pick a letter to guess the word."
+		@letter = gets.chomp.downcase
+		@letters_array = [*?a..?z] #creates an array of the alphabet from a to z lowercase
```

Comment:
The `*` in `[*?a..?z]` is the _destructuring operator_. It's a powerful tool, but I'm surprised to see it here. At this point, I don't think many folks in class would recognize what's happening. I'm really glad you put the comment in the line here as it adds clarity to what the array destructuring is doing.

You could express this in different ways, too, like `('a'..'z').to_a`, which might prove cognitively more clear.